### ISSUE TITLE
[BUG] Can't place a chest above a chestshop if display is none

### ISSUE THREAD

**Issue by Plattio** [snowgears/shopbugs#373]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Other unexpected behaviour

### Environment

```markdown
- Server Version: 1.19 (git-Paper-61)
- Shop Version: 1.8.3.8
```

### Server startup log

Not relevant

### Shop config files

https://pastebin.com/CHAkQFPb

### Bug description

Can't place a chest/any block above a chestshop if display is none. (displayNameTags and displayType)
GIF: https://imgur.com/a/oY0Wn3e

### Steps to reproduce

1. Place a chestshop sign on a chest
2. Try to place a block / chest above the chestshop with display none

### Expected behaviour

I can place any block above the chestshop.

### Actual behaviour

I can't place any block above the chestshop.

### Other information

GIF: https://imgur.com/a/oY0Wn3e

**Comment by snowgears** [comment]
Can you please test with the latest version of Shop and let me know if this is still a bug for you?

**Comment by Plattio** [comment]
> Can you please test with the latest version of Shop and let me know if this is still a bug for you?

Bug is still on the latest version

**Comment by snowgears** [comment]
I am testing on the latest version (1.8.4.1) in the same way shown in the GIF. I am able to place a chest above a shop when the display type is NONE and it will not let me place one when the display is any other type. Can you please try testing with just shop installed to make sure it is not a plugin conflict?

**Comment by Plattio** [comment]
Hi, still not possible with just Shop on the server.
![image](https://user-images.githubusercontent.com/65920067/213841083-d07a6872-795c-42ad-ad0e-b8f7b3be300a.png)
![image](https://user-images.githubusercontent.com/65920067/213841111-533f4403-f11b-4c15-a9b4-9992ce5916be.png)

**Comment by snowgears** [comment]
Thank you for the clarification!

I was testing with a shop that was display type NONE that was set via rotating displays. I see now that when the default is set to NONE this occurs.

Fixed in 1.8.4.2 (in progress)!
